### ISSUE TITLE
"a fatal error has occured" message while fencer maids was fighting mobs

### ISSUE THREAD

**Issue by abidoang** [KuroNeko87/littleMaidMobX#1]
Sometimes when the fencer maids are in action fighting some mobs , the world got forced to exit with the messages "A fatal error has occurred" (I think it occurred when they got damaged by the mobs, but it's not always). Here's the log: http://pastebin.com/nUNahKVV

**Comment by KuroNeko87** [comment]
could i have the actual crashlog?

**Comment by abidoang** [comment]
There's no crash log (I actually searched it in the crash report folder but there's nothing). The last time I tried after the maid killed a zombie, the world shut down and then it shows a message "a fatal error has occurred, this connection is terminated" (I tested it in a singleplayer world and it brought me back to the menu.
This is the log for the last test I did: http://pastebin.com/LnfKsaD4
Notice that after little maid was mentioned in the log, all other mods that was loaded is stopping.
